loki/tools/files.py

```diff
@@ -242,3 +244,116 @@ def load_module(module, path=None):
         # If module caching interferes, try again with clean caches
         invalidate_caches()
         return import_module(module)
+
+
+def write_env_launch_script(here, binary, args):
+    """
+    Utility method that is used for regression tests that require
+    activating an environment file before running :data:`binary`.
+
+    This writes a simple script of the form
+
+    .. code-block::
+       source env.sh
+       bin/<binary> <args>
+       exit $?
+
+    Parameters
+    ----------
+    here : pathlib.Path or str
+        The directory in which the script is created
+    binary : str
+        The name of the binary
+    args : list of str
+        List of arguments to pass to the binary
+
+    Returns
+    -------
+    pathlib.Path
+        The path to the created script file
+    """
+
+    script = Path(here/f'build/run_{binary}.sh')
+    script.write_text(f"""
+#!/bin/bash
+
+source env.sh >&2
+bin/{binary} {' '.join(args)}
+exit $?
+    """.strip())
+    script.chmod(0o750)
+
+    return script
+
+
+def local_loki_setup(here):
+    """
+    Utility method that is used to determine paths for injecting the
+    currently running source code of Loki into an
+    [ecbundle](https://github.com/ecmwf/ecbundle)-based worktree This
```

Comment:
Sorry, got caught out by the annoying RST vs. MD syntax differences for link formatting...

Also, empty line after the first sentence improves formatting in the HTML output.
```suggestion
    `ecbundle <https://github.com/ecmwf/ecbundle>`_-based worktree.
    
     This
```